.vscode/launch.json

```diff
@@ -215,29 +215,15 @@
         {
             "type": "node",
             "request": "launch",
-            "name": "UTAM Run Test - iOS",
+            "name": "UTAM Run Test",
```

Comment:
UTAM Run command turned out to be platform agnostic so no need to have 2 sets defined here with exactly the same params.